include/stdint.h

```diff
@@ -9,8 +9,12 @@
 #include <limits.h>
 
 #if __LP64__
+
+/*! Expands to 64. System word size */
 #define __WORDSIZE 64
```

Comment:
These are internal values that are used for selected some definitions later (e.g. uintptr max)